### PR TITLE
[bitnami/*] test: :white_check_mark: Support restricted PSA

### PR DESCRIPTION
**Pull request by javsalgar**
### Description of the change

This PR adapts the ginkgo tests to be run in a PSA restricted environment.

Result:

```
Ran 1 of 1 Specs in 39.269 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 0 Skipped
PASS
```

### Applicable issues

- fixes #

### Checklist

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [ ] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [ ] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [ ] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: .vib/mariadb/ginkgo/mariadb_suite_test.go [hunk 1]
```diff
@@ -40,6 +40,21 @@ func TestMariaDB(t *testing.T) {
...
+			Drop: []v1.Capability{"ALL"},
+		},
+		SeccompProfile: &v1.SeccompProfile{
+			Type: "RuntimeDefault",
+		},
+	}
+
+	*securityContext.Privileged = false
+	*securityContext.AllowPrivilegeEscalation = false
+	*securityContext.RunAsNonRoot = true
```
Comment (javsalgar) [inline .vib/mariadb/ginkgo/mariadb_suite_test.go:57]: It wouldn't allow me to put directly the values in the struct declaration above.

Comment (juan131) [reply .vib/mariadb/ginkgo/mariadb_suite_test.go:57]: I have a trick for you that's going to blow your mind 🤯 
```golang
	securityContext := &v1.SecurityContext{
		Privileged:               &[]bool{false}[0],
		AllowPrivilegeEscalation: &[]bool{false}[0],
		RunAsNonRoot:             &[]bool{true}[0],
		Capabilities: &v1.Capabilities{
			Drop: []v1.Capability{"ALL"},
		},
		SeccompProfile: &v1.SeccompProfile{
			Type: "RuntimeDefault",
		},
	}
```

**Review by juan131 — CHANGES_REQUESTED** [review]

**Review by juan131 — DISMISSED** [review]
LGTM

**Review by juan131 — APPROVED** [review]
LGTM

**Pull request merged**